src/namespacedActions.js

```diff
@@ -0,0 +1,64 @@
+import Actions from './actions'
+
+export default class NamespacedActions extends Actions {
```

Comment:
Yeah, I thought about that and it's probably a good idea. I wanted to think a bit more about possible changes to the the api for that though, but I think it's something we should do before final release.